CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
+- [otelgrpc] refactor otelgrpc to use grpc.StatsHandler. (#3002)
```

Comment:
This is not a refactoring[^1] as it changes the output.
Can you describe the changes more precise?

[^1]: Refactoring is a disciplined technique for restructuring an existing body of code, altering its internal structure without changing its external behavior.

Reply:
Done